src/Contexts/NetworkManagerContext.tsx

```diff
@@ -56,7 +56,8 @@ export type TransitMessage = {
 
 export type TransactionStatus =
   | "Initializing Transfer"
-  | "In Transit"
+  | "Transfer from Source"
```

Comment:
Statuses do not look clear. 
Please, draw them on a [diagram](https://miro.com/app/board/uXjVO_9kd3o=/?moveToWidget=3458764529262498598&cot=14) first.

Reply:
Don't have permissions to update it on my Cere account.

Reply:
Granted.

Reply:
Updated

Reply:
Please, add the events when statuses are changing for EVM <-> Cere flows.

Reply:
Updated miro diagram with technical details